policy_definitions/tagging/policy_definition_qby_require_tags.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "QBY-Require-Tag",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "apiVersion": "2021-06-01",
+    "properties": {
+        "policyType": "Custom",
+        "mode": "All",
```

Comment:
I think due to the [mode `All`](https://learn.microsoft.com/en-us/azure/governance/policy/concepts/definition-structure#mode) and no other filters, you are denying any resource, that doesn't support tags in general.
@QBY-ChristianHartmann do you know more?

Reply:
To avoid this, we should use INdexed. Then we need another POlicy to require tags on resourceGroups with the All mode.

Reply:
Create two definitio polices, one for RG with mode ALL and other for resources with mode INDEXED.